js/console.go

```diff
@@ -32,10 +33,24 @@ type Console struct {
 	Logger *log.Logger
 }
 
+// NewConsole creates a console with the standard logrus logger.
```

Comment:
Thanks for documenting this :) one less warning in golint but maybe we can just make them unexported?